tests/integration/conftest.py

```diff
@@ -587,7 +621,7 @@ def qtest_test_runs(self):
         """A list of qTest swagger_client detailed test run models associated with the given test case.
 
         Returns:
-            list(swagger_client.TestRunWithCustomFieldResource)
+            [swagger_client.TestRunWithCustomFieldResource)]
```

Comment:
The JetBrains IDE does not recognize "list(*)" as a valid docstring data type. Fixed.